### PR TITLE
Make UI more keyboard navigable

### PR DESCRIPTION
**Pull request by delonnewman**
Note: I'm not sure why `pane.hide()` in [javascript/uber_search.js:111](https://github.com/delonnewman/uber_select/blob/master/javascript/uber_search.js#L111) causes the pane to show.

### CODE REVIEW

File: javascript/output_container.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 var OutputContainer = function(options){
   options = $.extend({}, options)
-  var view         = this.view = $('<span class="selected_text_container" tabindex="0"></span>')
+  var view         = $('<span class="selected_text_container" tabindex="0"></span>')
```
Comment (njakobsen) [inline javascript/output_container.js:3]: Why this change?

File: javascript/pane.js [hunk 2]
```diff
@@ -25,13 +25,17 @@ function Pane(options){
...
     $(options.trigger).on('keyup', function(event){
       if ($(options.trigger).hasClass('disabled')) { return }
 
-      if (event.which == 13 || event.which == 32){
+      if (event.which === 32 || event.which === 40){
         context.show()
         return false
+      } else if (event.which === 13) { // toggle pane when enter is pressed
+        if (isOpen) context.hide()
+        else        context.show()
```
Comment (njakobsen) [inline javascript/pane.js:37]: Let's make a `toggle` function and hide the details in there.

File: javascript/uber_search.js [hunk 3]
```diff
@@ -106,7 +108,7 @@ var UberSearch = function(data, options){
     }
 
     setValue(valueFromResult(this))
-    pane.hide()
+    // pane.hide() FIXME: Not sure why this is causing the pane to show
```
Comment (njakobsen) [inline javascript/uber_search.js:111]: Let's work through this on a call.

File: javascript/pane.js [hunk 4]
```diff
@@ -25,13 +25,17 @@ function Pane(options){
       context.show()
     })
 
-    // Show the pane if the user was tabbed onto the trigger and pressed enter or space
+    // Show the pane if the user was tabbed onto the trigger and pressed enter, space, or down arrow
     $(options.trigger).on('keyup', function(event){
       if ($(options.trigger).hasClass('disabled')) { return }
 
-      if (event.which == 13 || event.which == 32){
+      if (event.which === 32 || event.which === 40){
```
Comment (njakobsen) [inline javascript/pane.js:32]: Will we have an issue with the down arrow event being stopped (`return false`), when we're trying to use arrow keys to move down the list? How about the spacebar event being stopped while typing in the search input? I could imagine the spacebar event probably would be fine, but the down arrow may be an issue if there is no search field enabled. See 
https://github.com/culturecode/uber_select/blob/ebb8bd3f9a9350865221875cc78eca5ac12d3cdb/javascript/list.js#L10 and https://github.com/culturecode/uber_select/blob/ebb8bd3f9a9350865221875cc78eca5ac12d3cdb/javascript/uber_search.js#L55.

**Review by njakobsen — CHANGES_REQUESTED** [review]

File: javascript/output_container.js (hunk 1 above)
Comment (delonnewman) [reply javascript/output_container.js:3]: I added it in the last PR, but I handed noticed that view is already in the public API.

File: javascript/pane.js (hunk 4 above)
Comment (delonnewman) [reply javascript/pane.js:32]: When I was testing it didn't seem to be an issue, but perhaps you're aware of something I'm not on this one.
